include/amici/defines.h

```diff
@@ -1,20 +1,61 @@
 #ifndef AMICI_DEFINES_H
 #define AMICI_DEFINES_H
 
-#include <cmath>
+#if !defined(_USE_MATH_DEFINES)
+#define _USE_MATH_DEFINES
+#endif
+
 #include <functional>
 #include <string>
+#include <cmath>
+
+/* Math constants in case _USE_MATH_DEFINES is not supported */
```

Comment:
According to M$VC documentation, it should be supported, but here it didn't seem to work. Maybe because the first inclusion didn't have it defined. At least we have some fallback now...